packages/passport/README.md

```diff
@@ -39,8 +39,9 @@ yarn dev
 # Context: passport-sdk-sample-app in ./sdk-sample-app
 yarn dev
 
-# Alternatively, you can run both commands in one terminal via the below
-yarn workspace @imtbl/passport-sdk-sample-app dev & yarn workspace @imtbl/passport dev
+# Alternatively, you can run both commands in one terminal via the below concurrent yarn script
+# Context: passport-sdk-sample-app in ./sdk-sample-app
+yarn dev-with-sdk
```

Comment:
Since we combined `passport/sdk/README` and `passport/sdk-sample-app/README` into `/passport/README`, the commands mentioned in this file would fail, as no `/passport/package.json` exists.

Can we update these commands to include the workspace please? E.g:
```
yarn workspace @imtbl/passport dev
...
yarn workspace @imtbl/passport-sdk-sample-app dev
...
yarn workspace @imtbl/passport-sdk-sample-app dev-with-sdk
```

Reply:
The context that's mentioned above specifies the context where the commands needs to be run. I've added a link to what this means at the start of the quick start section. The context tells you where to run it, and you decide how to then run it within that context, since devs may prefer their own way, either cding or yarn workspace.

For eg, with that above command, if the command needs to run in the sample app context as mentioned, then devs can do either 
```bash
yarn workspace @imtbl/passport-sdk-sample-app dev-with-sdk
```
or
```bash
cd packages/passport/sdk-sample-app
yarn dev-with-sdk
```

I've chosen this approach since it allows for flexibility rather than forcing a specific way of doing things, that can be annoying. I also went with this since I've seen devs using both ways rather than one way being much more consistent for all/most devs

Reply:
Another approach would be to just set up these dev commands in the root package.json for each package, but then that would bloat that with a ton of scripts and I felt that wouldn't have been the best approach.